src/test/scala/uk/gov/nationalarchives/tdr/api/routes/FileMetadataRouteSpec.scala

```diff
@@ -161,8 +160,8 @@ class FileMetadataRouteSpec extends TestContainerUtils with Matchers with TestRe
       val fileOneId = UUID.fromString("51c55218-1322-4453-9ef8-2300ef1c0fef")
       val fileTwoId = UUID.fromString("7076f399-b596-4161-a95d-e686c6435710")
       val fileThreeId = UUID.fromString("d2e64eed-faff-45ac-9825-79548f681323")
-      utils.addFileProperty("newProperty1", propertyGroup = OptionalMetadata)
-      utils.addFileProperty("existingPropertyUpdated1", propertyGroup = OptionalMetadata)
+      utils.addFileProperty("newProperty1", propertyType = "Supplied")
```

Comment:
Is there a reason for adding these properties?

Reply:
I had to set `propertyType` because we are filtering out `editable` fileProperties and by default the `utils.addFileProperty` method sets the `propertyType` to `System` which means it not editable and test fails.